compiler/rustc_infer/src/infer/glb.rs

```diff
@@ -30,6 +30,10 @@ impl<'tcx> TypeRelation<'tcx> for Glb<'_, '_, 'tcx> {
         "Glb"
     }
 
+    fn intercrate(&self) -> bool {
+        self.fields.infcx.intercrate
```

Comment:
can you change that to
```suggestion
        assert_eq!(self.fields.infcx.intercrate, false);
        false
```